scripts/format/format-src.sh

```diff
@@ -3,3 +3,4 @@ cd "$(dirname ${BASH_SOURCE[0]})"
 
 ./clang-format-script.sh ../../ros_ws/ file
 
+autopep8 --in-place --recursive --aggressive --aggressive ../../ros_ws/
```

Comment:
I would add a newline here as by unix convention (see https://robots.thoughtbot.com/no-newline-at-end-of-file)